code/modules/mob/living/carbon/human/species/wryn.dm

```diff
@@ -3,14 +3,8 @@
 	name_plural = "Wryn"
 	icobase = 'icons/mob/human_races/r_wryn.dmi'
 	deform = 'icons/mob/human_races/r_wryn.dmi'
-	blacklisted = TRUE
 	language = "Wryn Hivemind"
 	tail = "wryntail"
-	punchdamagelow = 0
-	punchdamagehigh = 1
-	speed_mod = 1
-	warning_low_pressure = -300
-	hazard_low_pressure = 1
```

Comment:
These pressure modifiers are spaceproofness that we're getting rid of ala Fox McCloud's push on the upstream.